api/Badges.json

```diff
@@ -41,7 +41,7 @@
             "children": [
                 {
                     "tag": "path",
-                    "color": "rgb(255, 0, 103)",
+                    "color": "rgb(255, 0, 0)",
```

Comment:
Please update this to use the hexadecimal format of this colour, we believe that hexadecimal colours matter and that when applicable, use them.